src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,368 @@
+    "app_name": "Pocket Sync",
+    "new_firmware": "发现新的Pocket固件 v{version}",
+    "update_available": "软件有更新! {version}"
+  },
+  "app": {
+    "app_name": "Pocket Sync",
+    "connect_button": "连接到Pocket",
+    "not_pocket_tip": "目录无法识别, 请确认SD卡已经由Pocket完成初始化"
+  },
+  "clean_files": {
+    "close_button": "关闭",
+    "delete_files_button": "删除文件",
+    "files_found": "共找到{count, plural, =0 {0个可清理文件} one {# 个可清理文件} other {# 个可清理文件} }",
+    "title": "清理文件"
+  },
+  "core_info": {
+    "author": "作者",
+    "controls": {
+      "back": "返回列表",
+      "install": "安装",
+      "required_files": "必要文件",
+      "uninstall": "卸载",
+      "update": "更新"
+    },
+    "inputs": "按键提示",
+    "modal": {
+      "close": "关闭",
+      "input_core": "核心",
+      "no_settings_found": "未发现设置文件!",
+      "reset_all": "重置所有设置"
+    },
+    "requires_license": {
+      "all": "请注意!运行该核心需要取得许可授权",
+      "jotego": "JOTEGO betas需要JTBETA许可文件才能运行，可以通过JOTEGO的Patreon页面获取(https://www.patreon.com/jotego)。 \n 如果您取得了许可文件，请将jtbeta.zip放置在SD卡根目录，系统将自动根据必要文件的处理规则进行处理。"
+    },
+    "not_in_inventory": "核心{coreName}尚未加入清单",
+    "platform": {
+      "edit": "编辑",
+      "wikipedia": "维基百科"
+    },
+    "replaced": "此核心已经由<1>{replacementCore}</1>核心替代",
+    "platforms": "平台",
+    "please_wait": "请稍后，检查文件中...",
+    "release_date": "发布日期",
+    "releases": {
+      "release_history": "版本发布历史",
+      "show_previous": "显示之前的版本发布历史信息"
+    },
+    "required_files": "必要文件",
+    "required_files_count": "{count} / {total} 文件",
+    "settings": "配置",
+    "sponsor": "Sponsor",
+    "supports": "Supports",
+    "supports_items": {
+      "cartridges": "卡带",
+      "dock": "Analogue 底座", 
+      "dock_dac": "Analogue DAC解码器", 
+      "save_states_and_sleep": "即时存档/睡眠"
+    },
+    "url": "网页地址",
+    "version": "版本"
+  },
+  "core_info_required_files": {
+    "close": "关闭",
+    "download_all": "下载全部",
+    "no_link_tip": "请进入配置页面查询必要文件的更多选项",
+    "skip_alternates_label": "跳过次要文件",
+    "title": "必要文件",
+    "file_status": {
+      "wrong": "可更新",
+      "downloadable": "可下载",
+      "not_in_archive": "未入库",
+      "ok": "已下载",
+      "unknown": "未知",
+      "at_root": "已存在于根目录"
+    }
+  },
+  "cores": {
+    "controls": {
+      "category": "类型",
+      "refresh": "刷新",
+      "search": "搜索",
+      "updatable": "可更新"
+    },
+    "install_tip": "您也可以通过拖拽ZIP压缩包到窗口内的方式安装核心或其他内容",
+    "installed": "已安装({count})",
+    "not_installed": "可安装({count})"
+  },
+  "error": {
+    "report": "您可以通过GitHub报告错误信息（含错误信息和错误发生的描述性文字）",
+    "retry": "重试",
+    "title": "错误"
+  },
+  "fetch": {
+    "controls": {
+      "add_fetch_item": "添加新的同步",
+      "refresh": "刷新"
+    },
+    "copying": "正在复制文件...",
+    "fetch_button": "同步",
+    "file_count": "{count} / {total, plural, one {# 文件} other {# 文件}}",
+    "loading": "载入中……",
+    "new": {
+      "add_extension": "添加",
+      "add_fetch": "添加同步",
+      "archive_name": "来源名称: <i>https://archive.org/details/[详细地址，例如gameboyroms_2019xx]</i>",
```

Comment:
Better to avoid any explicit mention of roms - I've been using "library images" or something as the example

Reply:
Understood. I removed the sensitive words